README.md

```diff
@@ -33,11 +33,13 @@ Usage: pa11y-ci [options]
 
 Options:
 
-  -h, --help           output usage information
-  -V, --version        output the version number
-  -c, --config <path>  the path to a JSON or JavaScript config file
-  -s, --sitemap <url>  the path to a sitemap
-  -j, --json           Output results as JSON
+  -h, --help                      output usage information
+  -V, --version                   output the version number
+  -c, --config <path>             the path to a JSON or JavaScript confi   file
+  -s, --sitemap <url>             the path to a sitemap
+  -f, --sitemap-find <pattern>    a pattern to find in sitemaps. Use wit   --sitemap-replace
+  -r, --sitemap-replace <string>  a replacement to apply in sitemaps. Us   with --sitemap-find
+  -j, --json                      Output results as JSON
```

Comment:
looks like above should be:
Use with --sitemap-replace
Use with --sitemap-find
?